test/e2e/webdriver/driver.js

```diff
@@ -19,12 +19,14 @@ const cssToXPath = require('css-to-xpath');
  */
 function wrapElementWithAPI(element, driver) {
```

Comment:
Is this method used with a delay anywhere? If not, consider reverting changes to this method.

Reply:
Take another approach now, Hope this will work better 🙏  https://github.com/MetaMask/metamask-extension/pull/19372/commits/6e7ef932c6a48386e7f717ca0d1db82f73f213f9